services/AUTOMATIC1111/Dockerfile

```diff
@@ -1,14 +1,10 @@
 # syntax=docker/dockerfile:1
 
-FROM alpine/git:2.36.2 as download
+FROM alpine:3.17 as download
 
-SHELL ["/bin/sh", "-ceuxo", "pipefail"]
```

Comment:
it is not possible to have shell config at all? no other way to define it?

Reply:
podman will ignore it completely, at least it does for me.

With that said, I think changing the statement to && should cover `e`, docker/podman already print the full RUN command, so that should cover `x`. 

For pipefail and `u`, I think I can only suggest making complicated RUN command a separate script, like the `clone.sh` one.